pkg/lib/k8s/pod.go

```diff
@@ -333,3 +333,20 @@ func (c *Client) StalledPods() ([]kcore.Pod, error) {
 
 	return stalledPods, nil
 }
+
+func Tolerations() []kcore.Toleration {
```

Comment:
nit: I think it might be cleaner to keep the `k8s` package not cortex-specific. Can this be easily moved to another package?